app/code/Magento/Sales/Test/Mftf/Test/AdminMassOrdersCancelCompleteAndClosedAPITest.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminMassOrdersCancelCompleteAndClosedAPITest">
```

Comment:
I would suggest removing the API keyword from the test name. Despite the fact we create entities via API, we test the admin UI behavior eventually (not the API). For API we have a different type of tests. 

```suggestion
    <test name="AdminMassOrdersCancelCompleteAndClosedAPITest">
```

Reply:
Hi @rogyar, I have renamed the test. Thank you!